theme/templates/pages/homepage.html

```diff
@@ -148,6 +147,21 @@ <h4>Share with colleagues</h4>
                         </div>
```

Comment:
I think it is better to remove the subject ```You``` on line 142 to make it ```Choose who has access ...``` to be consistent with descriptions for other steps.